resources/testing/README.md

```diff
@@ -0,0 +1,74 @@
+
+## Testing and local environment automation - introduction
+
+Within this directory is an automation script that has been created to make setting up and resetting a local database environment faster for either new starters or between end to end tests. The script is simple, it looks for the subdirectories contained within and reports them back as an array. This first step is just a terminal GUI to allow the selection of the service you wish to use.
+
+Within each of the array elements, the script looks for the presence of a further script called automate-queries in each of the directories and that script does the actual automation and setup.
+Each element in the array has an assigned number and by selecting that number, the automate-queries script loads any common resets that are not specific to that scheme or use-case before then loading the list of specific queries needed for that use-case.
+As an example, all SFI end to end tests currently use the same database setup, most of the tables are truncated the same regardless of it being SFI22 or SFI23, but the data being loaded in might differ so in this case all the database-resets are performed initially, then SFI22 would load in its data if selected, whereas SFI23 would load in its own version of that data if selected.
+
+In all instances, the PGAdmin component can be a totally clean instance as the script will create the server group and databases before then performing the queries. This gives a greater flexibility in cases such as pgAdmin being used in Docker instead of a local installation and allows for the user to do a full volume teardown between tests if needed.
+
+## Installation of PostgreSQL in your WSL
+1.To install PostgreSQL on WSL (ie. Ubuntu):
+Open your WSL terminal (ie. Ubuntu).
+Update your Ubuntu packages: sudo apt update
```

Comment:
I would separate out this command line into its own row and add it as a quote / emphasis - e.g.

![image](https://github.com/DEFRA/ffc-pay-core/assets/60177449/20bcc8da-88a9-4c2b-87cb-6cc805b56578)

Same goes for anywhere you make a comment on a command line prompt to be ran by the user.

Reply:
One outstanding command line to emphasize - sudo apt update 
![image](https://github.com/DEFRA/ffc-pay-core/assets/60177449/81b7c591-5d91-4aac-aa23-4b1667791d4b)